dev-ruby/net-http-pipeline/net-http-pipeline-1.0.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+
+inherit ruby-fakegem
+
+DESCRIPTION="An HTTP/1.1 pipelining implementation atop Net::HTTP"
+HOMEPAGE="http://docs.seattlerb.org/net-http-pipeline/"
+
+LICENSE="MIT"
+SLOT="0"
```

Comment:
I'd prefer to use a SLOT here matching the semver scheme, so in this case SLOT="1". That makes it a bit easier to handle all the ruby versioning madness in Gentoo.